static/js/issuesTable.js

```diff
@@ -147,17 +201,38 @@ const IssuesTable = {
                             <th data-field="type">Type</th>
                             <th data-field="status">Status</th>
                             <th data-field="source_type">Source</th>
-                            <th data-field="assignee">Assignee</th>
-                            <th data-field="engagement">Engagement</th>
+                            <th data-field="assignee.name">Assignee</th>
+                            <th data-field="engagement.name">Engagement</th>
                             <th data-field="tags" data-events="tagsEvents" data-formatter="TagsFormatter.format">Tags</th>
-                            <th data-formatter="actionsFormatter" data-events="actionsEvents"></th>
+                            <th data-field="actions" data-formatter="actionsFormatter" data-events="actionsEvents"></th>
                         </tr>
                     </thead>
                     <tbody>
                     </tbody>
                 </table>
             </div>
         </div>
+
+        <div class="detail-container">
+                <small-issues-table
+                    v-show="selectedTicket"
+                    :issue="selectedTicket"
+                    :engagement="engagement"
+                    :pageNumber="pageNumber"
+                    :maxPageCount="maxPageCount"
+                    :ticket="selectedTicket"
+                    @updated="handleTicketChange"
```

Comment:
instead of this you can make your vue component accept the following syntax:
`v-model="selectedTicket"` so you can skip swriting handler :)

Reply:
I needed to perform not only the assignment but refresh the bootstrap table as well. In that case, I guess it is okay to use such event handler